frontend/Makefile

```diff
@@ -1,15 +1,21 @@
 SHELL = /bin/bash
 TAG ?= $(shell git describe --exact-match 2>/dev/null)
 COMMIT = $(shell git rev-parse --short=7 HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
+ARO_HCP_FRONTEND_IMAGE_BASE = ${RP_IMAGE_ACR}.azurecr.io/aro
```

Comment:
What is the value of `RP_IMAGE_ACR`?

The same comment applies than with https://github.com/Azure/ARO-HCP/pull/17/files#r1539370064